### PR TITLE
[IMP] website_form_project: improve description set from website form data

### PR DESCRIPTION
**Pull request by ravb-odoo**
In this PR, the formatting of adding descriptions to tasks created from the website form has been improved.

task-3459958

### CODE REVIEW

**Review by pkyriakou — COMMENTED** [review]
Thank you @ravb-odoo 
I left a few minor comments for you to look into. They apply also to the enterprise PR.

File: addons/website_form_project/controllers/main.py [hunk 1]
```diff
@@ -12,4 +15,25 @@ def insert_record(self, request, model, values, custom, meta=None):
             if visitor_partner:
                 values['partner_id'] = visitor_partner.id
 
-        return super().insert_record(request, model, values, custom, meta=meta)
+        res = super().insert_record(request, model, values, custom, meta=meta)
+        if (custom or meta) and model.model == 'project.task':
+            record = request.env['project.task'].sudo().browse(res)
+            custom = custom.replace('email_from', 'Email').replace(' :', ':')
+            _custom_label = "<h2>%s</h2>\n\n" % _("Other Information")  # Title for custom fields
```
[the remark this answers is not in the document]
Comment (ravb-odoo) [reply addons/website_form_project/controllers/main.py]: Hello @pkyriakou 
I have done with necessary changes.

Thank you:)

**Review by pkyriakou — COMMENTED** [review]
Thank you @ravb-odoo, I have no more remarks, LGTM

**Review by auon-odoo — CHANGES_REQUESTED** [review]
Same comments here. Thanks for your work!

File: addons/website_form_project/controllers/main.py [hunk 2]
```diff
@@ -12,4 +15,25 @@ def insert_record(self, request, model, values, custom, meta=None):
             if visitor_partner:
                 values['partner_id'] = visitor_partner.id
 
-        return super().insert_record(request, model, values, custom, meta=meta)
+        res = super().insert_record(request, model, values, custom, meta=meta)
+        if (custom or meta) and model.model == 'project.task':
```
Comment (auon-odoo) [inline addons/website_form_project/controllers/main.py:19]:
```suggestion
        res = super().insert_record(request, model, values, custom, meta=meta)
        if not (custom or meta) or model.model != 'project.task':
            return res
```

**Review by auon-odoo — APPROVED** [review]
Last comment. You can green the task when the changes are done and the runbot is green again :fire:

**Review by xavierbol — COMMENTED** [review]
Thanks for your work, I left 2 comments. 🙂

**Review by xavierbol — APPROVED** [review]
LGTM, thanks again for your work. 🙂 

robodoo r+

**Pull request closed without merging**